src/asm/macro.c

```diff
@@ -128,12 +128,17 @@ uint32_t macro_UseNewUniqueID(void)
 	return maxUniqueID;
 }
 
-void macro_ShiftCurrentArgs(void)
+void macro_ShiftCurrentArgs(int32_t count)
 {
 	if (!macroArgs)
 		error("Cannot shift macro arguments outside of a macro\n");
-	else if (macroArgs->shift != macroArgs->nbArgs)
-		macroArgs->shift++;
+	else if (count < 0)
+		error("Cannot %" PRId32 " times, that's a negative number!\n", count);
```

Comment:
The error message is missing a word, and isn't quite in the same tone as the rest of RGBASM's. I'd suggest `Cannot shift arguments by negative amount %PRId32`.

Reply:
Sorry that!